projects/plugins/boost/app/assets/src/js/pages/getting-started/GettingStarted.svelte

```diff
@@ -21,6 +22,10 @@
 		navigate( '/' );
 	};
 
+	onMount( () => {
+		recordBoostEvent( 'view_getting_started_page_in_plugin', {} );
```

Comment:
`recordBoostEvent` is an async function. I know this is `onMount` so it doesn't matter, but the other day you mentioned something about code being copy pasted later and that really stuck with me 😄

Reply:
46e15b6b5e adds a comment, and a `void` to explicitly throw away the unused promise. It is [the recommended approach to explicitly throwing away promises](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-floating-promises.md#options) when you are making the choice not to wait for them.